### PR TITLE
Fix snapshot mode

### PR DESCRIPTION
**Pull request by jimmy-huang**

### CODE REVIEW

**Comment by brianjjones** [comment]
LGTM +1

**Comment by jimmy-huang** [comment]
@brianjjones can you check if this patch will break ashell?  I think it might need the parser.

**Comment by brianjjones** [comment]
@jimmy-huang Good call, testing now.  @grgustaf  hold off on pulling this in until I confirm it's ok for ashell.  Thanks.

File: Makefile [hunk 1]
```diff
@@ -15,16 +15,19 @@ VARIANT ?= release
 # JerryScript options
 JERRY_BASE ?= $(ZJS_BASE)/deps/jerryscript
 EXT_JERRY_FLAGS ?= -DENABLE_ALL_IN_ONE=ON
+ifneq ($(DEV), ashell)
 ifeq ($(SNAPSHOT), on)
```
Comment (brianjjones) [inline Makefile]: This still won't work for 'make linux'.  I think this check / Flag set need to move to after ifeq ($(TARGET), linux).

**Comment by brianjjones** [comment]
LGTM thanks +1

File: Makefile [hunk 2]
```diff
@@ -36,6 +39,10 @@ PRINT_FLOAT ?= off
 # MAKECMDGOALS is a Make variable that is set to the target your building for.
 TARGET = $(MAKECMDGOALS)
 
+ifeq ($(TARGET), linux)
+	SNAPSHOT = off
+endif
```
Comment (grgustaf) [inline Makefile]: This block doesn't catch the "make BOARD=linux" case. (See line 58.) That will still try to build with snapshot.

Comment (jimmy-huang) [inline Makefile]: updated with check for BOARD=linux

**Comment by grgustaf** [comment]
+1

**Pull request merged**